pinc/page_controls.inc

```diff
@@ -75,24 +68,19 @@ function get_page_data()
 
     $page_browser_data = [
         "projectid" => $projectid,
-        "pages" => $images,
+        "pages" => $page_names,
         "currentPage" => $page,
         "currentRound" => $round_id,
         "text" => $text,
         "heading" => $heading,
         "errorMessage" => $error_message,
+        "imageUrl" => "$projects_url/$projectid/$page"
```

Comment:
Change `$projects_url/$projectid` to `$project->url` and drop the global for `$projects_url` altogether.

Reply:
good idea.